contents/security/cryptography.md

```diff
@@ -0,0 +1,81 @@
+# Introduction to Cryptography
+
+Authors: Dickson Tan
+
+# Overview
+
+Cryptography is the discipline of developing methods for secure communication. 
+Understanding cryptography wwill enable you to correctly use cryptography libraries and combine cryptographic primitives to match the security needs of your application.
+
+Modern cryptography involves developing methods to achieve the following:
```

Comment:
`Modern cryptography techniques involves developing methods to achieve the following objectives` ?